src/expression-engine/models/content/resolver.js

```diff
@@ -103,6 +103,21 @@ export default {
     body: ({ body, legacy_body }, _, { models }) => models.Content.cleanMarkup(body || legacy_body),
     description: ({ description }) => description,
     ooyalaId: ({ video }) => video,
+    video: ({ video }) => {
+      // Todo: should these be stored as ENV vars?
```

Comment:
these are public player codes from ooyala. you can find them now as part of the video embed.